Gruntfile.js

```diff
@@ -43,8 +43,8 @@ module.exports = function(grunt) {
 
     _accept_runner: '_mocha',
     _accept_args: '-A -u exports --recursive -t 10000 ./test/setup_accept.js ./test/accept',
-    accept: ['mongo ./test/setup_mongo.js', '<%= _accept_runner %> <%= _accept_args %>'],
-    accept_cover: ['mongo ./test/setup_mongo.js', 'istanbul cover --dir cov-accept <%= _accept_runner %> -- <%= _accept_args %>'],
+    accept: ['<%= _accept_runner %> <%= _accept_args %>'],
```

Comment:
Since these tasks should just work outside wendy also, then something might need to be done here. If some sort of Mongo setup is required, then I'd suggest either of the following:

* See if your `setup_mongo_wendy.js` can be used similar to (or as part of?) `setup_accept.js` on line 45 above
* Alternatively create a new task here called `accept_wendy` or something, and leave the existing ones alone